atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/AbstractDbWriteTable.java

```diff
@@ -119,6 +122,35 @@ public void putSentinels(Iterable<Cell> cells) {
     }
 
     @Override
+    public void update(Cell cell, long ts, byte[] oldValue, byte[] newValue) {
+        String prefixedTableName = prefixedTableNames.get(tableRef);
+        Object[] args = new Object[] {
+                cell.getRowName(),
+                cell.getColumnName(),
+                ts,
+                newValue,
+                cell.getRowName(),
+                cell.getColumnName(),
+                ts,
+                oldValue
+        };
+        String sqlString = "/* UPDATE (" + prefixedTableName + ") */"
+                + " UPDATE " + prefixedTableName + ""
+                + " SET row_name = ?, col_name = ?, ts = ?, val = ?"
+                + " WHERE row_name = ?"
+                + " AND col_name = ?"
+                + " AND ts = ?"
+                + " AND val = ?";
+        int updated = ((PalantirSqlConnection) conns.get()).updateCountRowsUnregisteredQuery(sqlString,
+                args);
+        if (updated == 0) {
+            // right now we don't know what's actually in the db :-(
+            throw new CheckAndSetException(cell, tableRef, oldValue, ImmutableList.of());
+        }
+        // TODO check > 1
```

Comment:
nit: Can we flesh this out or remove it?  Upon encountering this in the code, I'm not really sure what is intended.